src/components/partnership/partnership.service.ts

```diff
@@ -447,6 +529,29 @@ export class PartnershipService {
       throw new NotFoundException('Could not find partnership');
     }
 
+    let mouStart = null;
+    let mouEnd = null;
+
+    // if user has access to project mou and there is no partnership override
+    if (result.projectMouStart && result.mouStartOveride !== null) {
+      mouStart = result.mouStartOverride ?? result.projectMouStart;
+    }
+    if (result.projectMouEnd && result.mouEndOverride !== null) {
+      mouEnd = result.mouEndOverride ?? result.projectMouEnd;
+    }
+
+    let canReadMouStart = false;
+
+    if (result.canReadProjectMouStart && result.canReadMouStartOverride) {
+      canReadMouStart = true;
+    }
```

Comment:
FYI
```suggestion
    const canReadMouStart = result.canReadProjectMouStart && result.canReadMouStartOverride;
```

Reply:
lol. I was thinking I had more complexity but then decided against it and never cleaned it up. thanks for the catch.